Examples/SnowplowDemo/SnowplowDemo/ViewController.m

```diff
@@ -157,10 +159,24 @@ - (SPTracker *) getTrackerWithUrl:(NSString *)url_
         [builder setBase64Encoded:false];
         [builder setSessionContext:YES];
         [builder setSubject:subject];
+        [builder setGlobalContextGenerators:@{
+            @"globalContextTag1": [self getGlobalContext]
+        }];
     }];
     return tracker;
 }
 
+- (SPGlobalContext *)getGlobalContext {
```

Comment:
Same comment here if you take the idea forward of adding a "simple" static global context.